compiler/rustc_target/src/asm/m68k.rs

```diff
@@ -0,0 +1,77 @@
+use super::{InlineAsmArch, InlineAsmType};
+use rustc_macros::HashStable_Generic;
+use rustc_span::Symbol;
+use std::fmt;
+
+def_reg_class! {
+    M68k M68kInlineAsmRegClass {
+        reg,
+        reg_ptr,
```

Comment:
I would recommend the following classes instead:
- `reg`: data or address register (`r`)
- `reg_data`: data register (`d`)
- `reg_addr`: address register (`a`)